src/bin/iq/main.rs

```diff
@@ -0,0 +1,302 @@
+                                        iq.server.clone(),
+                                        res.url_results.report_html_url,
+                                    );
+
+                                    process::exit(1);
+                                }
+                                "Warning" => {
+                                    print_iq_policy_violations(res.policy_report_results, &parser);
+                                    print_iq_summary(
+                                        CONSTRUCTION,
+                                        style(
+                                            "Barnacles! Warnings have been detected in your scan.",
+                                        )
+                                        .yellow()
+                                        .bold(),
+                                        res.url_results.open_policy_violations,
+                                        iq.server.clone(),
+                                        res.url_results.report_html_url,
+                                    );
+                                }
+                                "None" => {
+                                    print_iq_summary(
+                                        SHIP,
+                                        style(
+                                            "Smooth sailing! No policy issues found in your scan.",
+                                        )
+                                        .green()
+                                        .bold(),
+                                        res.url_results.open_policy_violations,
+                                        iq.server.clone(),
+                                        res.url_results.report_html_url,
+                                    );
+                                }
+                                _ => {
+                                    println!(
+              "{}", "The response from Nexus IQ Server did not include a policy action, which is odd"
+              );
+
+                                    process::exit(1);
+                                }
+                            }
+                        }
+                        Err(e) => {
+                            iq_bar.finish_with_message(format!(
+                                "{}{}",
+                                CROSS_MARK, "Error generating Nexus IQ Server results"
+                            ));
+                            error!("{}", e);
+
+                            println!("{}", e);
+
+                            process::exit(1);
+                        }
+                    }
+                }
+                Err(e) => {
+                    package_bar.finish_with_message(format!(
+                        "{}{}",
+                        CROSS_MARK, "Unable to obtain package list"
+                    ));
+                    println!("{}", e);
```

Comment:
IDEA is flagging this, but it I'm not sure it makes sense to me.

     `Error` doesn't implement `Display` (required by {})

The higher `Err(e)` shows a type of `Err(e) Box<dyn Error>`, so maybe that is why the higher one (line 168) is not getting flagged? 
Just asking for my own knowledge - not a blocker seems to me. Dunno if we exercise this code.

Reply:
This was code that was moved and git failed to track it, so it has been working previously. I'm going to create an issue to follow up with this specifically in a separate branch